### PR TITLE
docs: improve events, hooks and other SP docs

### PR DESCRIPTION
**Pull request by CarlosLeyvaAyala**
I started working on expanding the documentation and I've documented `writeLogs()` .

I'm not **totally** convinced about what I've done with the new formatting (although I don't think it looks bad, either), especially with the format I used in new_methods.md, so I would like to hear a second opinion

### CODE REVIEW

**Comment by CarlosLeyvaAyala** [comment]
Done for now.

I'll add more info when I learn more about SP and deal with standardizing format at another PR.

File: docs/skyrim_platform/native.md [hunk 1]
```diff
@@ -1,64 +1,118 @@
...
-  ```
+```typescript
+let isPlayerInCombat = pl.isInCombat();
+```
 
-- A list of original game types with documentation can be found here: https://www.creationkit.com/index.php?title=Category:Script_Objects
-- Calling functions from the original game is only available inside the `update` event handler (see below). If you try to do this in a different context, an exception will be thrown.
+A list of original game types with documentation can be found here: https://www.creationkit.com/index.php?title=Category:Script_Objects
+
+***WARNING***: Papyrus types and methods are only available inside [hooks][Hooks] or [new events][NewEvents] introduced by Skyrim Platform (except `tick`).
```
Comment (Pospelove) [inline docs/skyrim_platform/native.md:21]: Honestly, I think they are available only in papyrus event hook. In anim event hook not. Let's check it.

File: docs/skyrim_platform/new_events.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+Analogous to [OnObjectEquipped][OnObjectEquipped].
+
+```typescript
+import { on } from "skyrimPlatform";
+
+on("equip", (event) => {
+  const b = event.actor.getBaseObject();
+  const armor = Armor.from(event.baseObj);
+
+  if (!armor || armor.getSlotMask() !== 0x4 || !b) return
```
Comment (Pospelove) [inline docs/skyrim_platform/new_events.md:90]: missing semicolon here

File: docs/skyrim_platform/new_events.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+```typescript
+import { on } from "skyrimPlatform";
+
+on("unequip", (event) => {
+  if (event.actor.getFormID() !== Game.getPlayer()?.getFormID()) return
+
+  const w = Weapon.from(event.baseObj);
+
+  if(w)
```
Comment (Pospelove) [inline docs/skyrim_platform/new_events.md:109]: Please fix formatting
```ts
if (w) {
  // ...
}
```

File: docs/skyrim_platform/new_events.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+## unequip
+
+Called each time any `Actor` unequips an item.\
+Analogous to [OnObjectUnequipped][OnObjectUnequipped].
+
+```typescript
+import { on } from "skyrimPlatform";
+
+on("unequip", (event) => {
+  if (event.actor.getFormID() !== Game.getPlayer()?.getFormID()) return
```
Comment (Pospelove) [inline docs/skyrim_platform/new_events.md:105]: semicolon 2 :)

File: docs/skyrim_platform/new_events.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+Analogous to [OnEffectStart][OnEffectStart].
+
+***HINT***: If you are using cloaks to apply Magic Effects in your mod, you can drastically increase performance by applying a blank Magic Effect with [SPID][SPID] instead of cloaks and letting Skyrim Platform deal with the real code:
+
+```typescript
+import { on } from "skyrimPlatform";
+
+on("effectStart", (event) => {
+  const fx = Game.getFormFromFile(0x800, "my-mod.esp");
+  if (fx?.getFormID() !== event.effect.getFormID()) return
```
Comment (Pospelove) [inline docs/skyrim_platform/new_events.md:126]: and here

**Review by Pospelove — CHANGES_REQUESTED** [review]
Thanks! Some cosmetic changes requested, overall looks good

**Review by Pospelove — APPROVED** [review]

**Pull request merged**